.github/workflows/ruby.yml

```
@@ -0,0 +1,30 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: Ruby
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

```suggestion
    branches: [master]
  pull_request:
    branches: [master]
```


style nit: the rest of the file does not use extra spaces here (see: matrix)

---

why is there no commit button on this suggestion 🤔